core/templates/dev/head/galleries/gallery.html

```diff
@@ -21,7 +21,12 @@
   <script type="text/javascript">
     GLOBALS.LANGUAGE_CODES_AND_NAMES = JSON.parse(
       '{{LANGUAGE_CODES_AND_NAMES|js_string}}');
-    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse('{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    GLOBALS.CAROUSEL_SLIDES_CONFIG = JSON.parse(
+      '{{CAROUSEL_SLIDES_CONFIG|js_string}}');
+    {% if explorations_list %}
```

Comment:
Thanks for the update! I'm a little concerned about the instability, since the aim of the suggestion was to make the overall design simpler. Would you like to meet up at some point to chat about the design? It's possible that there may be finer points that I've missed.